### DISCUSSION TITLE
How can I transfer the account to  my company account?

### DISCUSSION THREAD

**Discussion by shameemz** [codemagic-ci-cd/codemagic-docs#946] [Q&A] [answered]
How can I transfer to my company account?

**Comment by K-Codemagic** [comment, marked as answer]
Hi, can you please elaborate?

**Reply by K-Codemagic** [reply]
Hi, can you please provide detailed information so I can help you with the process? Usually, you can add apps by creating a Team. However, I would like to understand your query and give you a precise answer.

**Reply by K-Codemagic** [reply]
Teams can be created to transfer apps from one account to another.

**Comment by shameemz** [comment]
Yeah. I found that by adding as teams.

On Mon, 19 Sept 2022 at 10:15, K-Codemagic ***@***.***> wrote:

> Hi, can you please provide detailed information so I can help you with the
> process? Usually, you can add apps by creating a Team. However, I would
> like to understand your query and give you a precise answer.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/codemagic-ci-cd/discussions/946#discussioncomment-3677824>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAEIGC2KSD2K33ETM5NGVJDV7AAH7ANCNFSM6AAAAAAQLFGC5Q>
> .
> You are receiving this because you authored the thread.Message ID:
> <codemagic-ci-cd/codemagic-docs/repo-discussions/946/comments/3677824@
> github.com>
>


-- 
Shameem
+971 561017733
